vevestaX/vevesta.py

```diff
@@ -789,46 +802,72 @@ def __EDAForPandas(self, fileName):
         [x.tick_params(axis='x', labelrotation=90) for x in fig.ravel()]
         plt.plot()
         plt.suptitle('Feature Histogram',fontsize=20)
-        plt.savefig(os.path.join(directoryToDumpData, FeatureHistogramImageFile), bbox_inches='tight', dpi=100)
+        plt.savefig(join(directoryToDumpData, FeatureHistogramImageFile), bbox_inches='tight', dpi=100)
         plt.close()
+        
+        #Probability Density Function
+        numericDataframe = self.__data.select_dtypes(include='number')  
+        if self.__ycolumn!=None:
```

Comment:
if y is regression what happens then i dont see the implementation of what we discussed